modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -351,11 +353,16 @@ private PartialNode startPartialNode(
         LongSupplier partitionIdleSafeTimePropagationPeriodMsSupplier
                 = () -> TestIgnitionManager.DEFAULT_PARTITION_IDLE_SYNC_TIME_INTERVAL_MS;
 
-        var replicaService = new ReplicaService(
+        MessagingService messagingServiceReturningToStorageOperationsPool = new JumpToExecutorByConsistentIdAfterSend(
                 clusterSvc.messagingService(),
-                hybridClock,
                 name,
-                threadPools.partitionOperationsExecutor()
+                message -> threadPoolsManager.partitionOperationsExecutor()
```

Comment:
It is weird that the pool is called a partial operation.
This looks like a common system pool, based on this usage pattern.

Reply:
This pool is used for processing operations on partitions, hence the name. The wrapper around MessagingService that is constructed here is also used by the components that deal with processing operations on partitions, so it returns the control to the same partition-operations pool. But there are parts of the system that don't use it; they might use another wrapper around MessagingService to return to that another pool (maybe operations with the Metastorage should work like this). So, it's not the system-wide pool, its usage is still restricted.